Deploy/uwp_jcadmu.ps1

```diff
@@ -648,10 +647,17 @@ function New-UWPForm {
         </StackPanel>
 
         </Grid>
+        <TextBlock Name="ElapsedTimeTextBlock"

```

Comment:
Should we remove this? I remember you mentioned it's used for debugging

Reply:
It's not adding anything for the customer, I'll remove it.